src/App.tsx

```diff
@@ -15,48 +16,132 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortFieldValues {
+  default = '',
```

Comment:
```suggestion
  default = 'default',
```
better to make more meaningful nami of field instead empty string

Reply:
![Screenshot 2023-07-03 at 13 27 44](https://github.com/mate-academy/react_list-of-goods-hooks/assets/83166880/27ce798e-2c56-4571-9b0f-69c471602424)

😀😀😀 The same task without ts)